runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobe.java

```diff
@@ -123,4 +127,25 @@ public void setTime(Instant time)
 	{
 		this.time = time;
 	}
+
+	public double getXpPerHour()
```

Comment:
Use Lombok `@Getter` and `@Setter` annotations

Reply:
Also, XP/hour is imo completely out of scope of the informations that XP globes should provide, for that, there is XP tracker.

Reply:
Should XP/hour be disable on default then?